roxctl/deployment/check/check_test.go

```diff
@@ -270,14 +275,35 @@ func (d *deployCheckTestSuite) createMockEnvironmentWithConn(conn *grpc.ClientCo
 
 func (d *deployCheckTestSuite) SetupTest() {
 	d.defaultDeploymentCheckCommand = deploymentCheckCommand{
-		file:               "testdata/deployment.yaml",
+		files:              []string{"testdata/deployment.yaml"},
 		retryDelay:         3,
 		retryCount:         3,
 		timeout:            1 * time.Minute,
 		printAllViolations: true,
 	}
 }
 
+func (d *deployCheckTestSuite) TestMultipleFiles() {
+	deployCheckCmd := d.defaultDeploymentCheckCommand
+	deployCheckCmd.files = []string{"testdata/deployment.yaml", "testdata/deployment2.yaml"}
+
+	conn, closeF, server := d.createGRPCMockDetectionService(testDeploymentAlertsWithoutFailure, []string{""})
+	defer closeF()
+	deployCheckCmd.env, _, _ = d.createMockEnvironmentWithConn(conn)
+
+	tablePrinter, err := printer.NewTabularPrinterFactory(defaultDeploymentCheckHeaders,
+		defaultDeploymentCheckJSONPathExpression).CreatePrinter("table")
+	d.Require().NoError(err)
+	deployCheckCmd.printer = tablePrinter
+	err = deployCheckCmd.Check()
+	d.Require().NoError(err)
+	var expectedBinary []byte
```

Comment:
You don't need to define this here, just use the declaration + assignment notation below (`:=`):
```
expectedBinary, err := os.ReadFile("testdata/testMultipleFilesExpectedYaml.yaml")
```

Reply:
I thought this would not work because err was already declared above, but it did work.